src/jit/hwintrinsiclistxarch.h

```diff
@@ -11,59 +11,69 @@
 // clang-format off
 
 #if FEATURE_HW_INTRINSICS
-//                  Intrinsic ID                   Function name                ISA   
+// **************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//                  Intrinsic ID                          Function name                            ISA        ival       SIMD size                                                                             instructions                                                                                       Category
+//                                                                                                                                            {TYP_BYTE,      TYP_UBYTE,     TYP_SHORT,     TYP_USHORT,    TYP_INT,       TYP_UINT,      TYP_LONG,      TYP_ULONG,     TYP_FLOAT,     TYP_DOUBLE}
+// **************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
 //  SSE Intrinsics          
-HARDWARE_INTRINSIC(SSE_IsSupported,             "get_IsSupported",              SSE)
-HARDWARE_INTRINSIC(SSE_Add,                     "Add",                          SSE)
+HARDWARE_INTRINSIC(SSE_IsSupported,                      "get_IsSupported",                        SSE,        -1,           0,               {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_Special)
```

Comment:
nit: "get_IsSupported" should start at least at column 70 (quotation mark) to make space for larger intrinsic names

Reply:
Thanks! will change.